spec/avHangoutsNotificationsSpec.coffee

```diff
@@ -46,7 +20,7 @@ mockSlackHangoutNotify = (routes_functions, channel, type, project) ->
   nock("https://api.gitter.im")
     .post("/v1/rooms/56b8bdffe610378809c070cc/chatMessages")
     .reply(200, {error: 'not_authed'})
-  nock('https://slack.com', allowUnmocked: true)
+  nock('https://slack.com', allowUnmocked: false)
```

Comment:
@tansaku yes, once i did this it seemed to resolve any remaining of those errors that intermittently poked up.